tasks/components/RoomList.tsx

```diff
@@ -101,6 +101,9 @@ export const RoomList = ({
   const [stateDeletionConfirmDialog, setDeletionConfirmDialogState] = useState(defaultState)
   const resetDeletionConfirmDialogState = () => setDeletionConfirmDialogState(defaultState)
 
+  const minRoomNameLength = 2
```

Comment:
```suggestion
  const minRoomNameLength = 1
```
https://github.com/helpwave/web/issues/281#issuecomment-1520767575